lib/src/model/library.dart

```diff
@@ -315,7 +316,7 @@ class Library extends ModelElement
 
   /// The real packageMeta, as opposed to the package we are documenting with.
   late final PackageMeta? packageMeta =
-      packageGraph.packageMetaProvider.fromElement(element2, config.sdkDir);
+      packageGraph.packageMetaProvider.fromElement(element as LibraryElementImpl, config.sdkDir);
```

Comment:
Do we need specifically `LibraryElementImpl` here, or `LibraryElement2`?
Can we use `.asElement2` instead?